app/routes/docs.notices.schema.mdx

```diff
@@ -75,46 +96,64 @@ This allows our validator script to identify which schema examples to compare. M
   </Tab>
 </Tabs>
 
-## Defining Schema
+### Quick Reference
```

Comment:
```suggestion
### Quick Schema Reference
```

Reply:
Why?

Reply:
Wanted to be as explicit as possible

Reply:
How about "JSON Schema Quick Reference"?